### ISSUE TITLE
[device-port] [scale] Huawei Y6

### ISSUE THREAD

**Issue by benclarkk** [Halium/projectmanagement#152]
Tree:  hallium 7.1

- [x] Create manifest  
- [x] Boot image and system image build successfully
- [ ] Device boots into rootfs, `usb: Manufacturer: GNU/Linux Device` appears in `dmesg` on host.
- [ ] LXC container starts and does not crash
- [ ] libhybris tests
  * [ ] test_gps
  * [ ] test_hwcomposer
  * [ ] test_lights
  * [ ] test_vibrator
  * [ ] test_wifi
  * [ ] test_sensors
  * [ ] test_audio
  * [ ] test_camera
  * [ ] test_input
  * [ ] test_recorder

trying to get kernel build to work, 
logs here: https://bin.disroot.org/?76451edb0ed685dc#Vhi7swCGwe8My3gJLH1ALmBrYoK4fvXUIt0pM0P6pLw=

something about __aeabi_unwind_cpp_pr0 (???)

(i think i've fixed it by just disabling aeabi in the defconfig, i hope it's not anything important)

**Comment by benclarkk** [comment]
notes here: https://cloud.disroot.org/s/pCbdC5mDcfT3ZRe
other stuff here: https://cloud.disroot.org/s/8QJMGrZ3opJ7b2o

**Comment by Bluppie05** [comment]
Yes lets go thats the device we want ubuntu touch/hallium on! Go on with porting it to that model phone. Let me know if I could possible do anything to help you make a port of hallium for the Y6, my email aress is bluppie05@gmail.com I have an Y6 device so maybe I can help testing or somthing. Kind regards, Bluppie05

**Comment by benclarkk** [comment]
@Bluppie05 which model of y6 have you got? i kinda gave up after a while bc my y6 has a smashed screen.

**Comment by Bluppie05** [comment]
I have an Huawei Y6 SCL-L21 which is running emui 3.1(Summary: I have the old model)

**Comment by benclarkk** [comment]
ah, i have scl-l01 but it's not too hard to port to both (?)

**Comment by Bluppie05** [comment]
Ahh ok, isn't that the almost exact same precedur?

**Comment by Bluppie05** [comment]
Or does it have really big hardware differences?

**Comment by Bluppie05** [comment]
Oh the hardware difference is that the SCL-L21 has 2GB of memory and the SCL01 has 1GB of memory. Is that a big deal?

**Comment by Bluppie05** [comment]
Do you have something like Skype, Discord, Telegram or any other messaging app? Because thats a little bit easier to communicate with than the github comments.

**Comment by benclarkk** [comment]
yea, @jamesisaweeb on tg

**Comment by doniks** [comment]
could you share your manifest? the links don't work

**Comment by radunanescu** [comment]
Any news? i kinda wanna make this for fun

**Comment by benclarkk** [comment]
@radunanescu my y6 got cucked a while ago, but if you want to figure out yourself, go ahead!

**Comment by Bluppie05** [comment]
BRO, Why didn't you respond to my messages?

**Comment by radunanescu** [comment]
Do you still have the manifest?

**Comment by benclarkk** [comment]
nah, i don't think it even worked in the first place. i put it somewhere on github but i think i deleted it in my annual spring cleaning.
